src/configparser.js

```diff
@@ -231,22 +237,39 @@ ConfigParser.prototype.writeAsync = function(file) {
     return writeFileAsync(file, out);
 }
 
-function parseLines(lines) {
+function parseLines(lines, options) {
+    options = options || {};
+	let transform = options.transform;
     let curSec = null;
+    let upperCaseKey = function(key){
+	    if (key === void 0 || key === null) return;

```

Comment:
There seems to be an inconsistent use of spaces and tabs, could you use 4 space indentation please ?